packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -46,6 +49,7 @@ type Order = AlphabeticalOrder | 'as-written';
 interface SortedOrderConfig {
   memberTypes?: MemberType[] | 'never';
   order: Order;
+  requiredFirst?: boolean;
```

Comment:
Sorry, I realize this is adding on more complexity after we already marked the issue as accepting PRs, but... what if people want to add required _last_? What do you think about changing this to `required?: 'first' | 'last'`?

Reply:
Yeah, I think its a good idea. I don't mind tacking it on in this PR.